CHANGELOG-1.2.md

```diff
@@ -0,0 +1,10 @@
+# Changelog since v1.1.0
+
+## Bug Fixes
+
+- Changes the API group name for the fake VolumeSnapshot object to "snapshot.storage.k8s.io" to be in-sync with the group name of the real VolumeSnapshot object. As a result, the generated interfaces for clientset and informers of VolumeSnapshot are also changed from "VolumeSnapshot" to "Snapshot". ([#123](https://github.com/kubernetes-csi/external-snapshotter/pull/123), [@xing-yang](https://github.com/xing-yang))
```

Comment:
Is this a breaking change? Will users that have previous yamls or client libraries need to be updated?

Reply:
This does not have impact on yaml files because the API object name and API group stays the same.
This does affect people who use clientset and informers of volume snapshots.

Reply:
Then we should have one more section at the top for breaking changes and call this out.

Reply:
Sure.